lib/localenv/localenv.go

```diff
@@ -463,6 +464,11 @@ func (env *LocalEnvironment) SiteOperator() (*opsclient.Client, error) {
 	return operator, trace.Wrap(err)
 }
 
+// LocalClusterOperator returns Operator for the local gravity cluster
+func (env *LocalEnvironment) LocalClusterOperator() (ops.Operator, error) {
```

Comment:
Why not just rename `SiteOperator()` to `LocalClusterOperator()` then? :) Too many changes?